default/AI/ResearchAI.py

```diff
@@ -71,379 +384,27 @@ def generate_research_orders():
-            except:
-                print "    Error: failed attempt to enqueued Tech: " + name
-                print "    Error: exception triggered and caught: ", traceback.format_exc()
-
-
-        print "\n\nAll techs:"
-        alltechs = fo.techs()  # returns names of all techs
-        for tname in alltechs:
-            print tname
-        print "\n-------------------------------\nAll unqueued techs:"
-        # coveredTechs = new_tech+completed_techs
-        for tname in [tn for tn in alltechs if tn not in tech_base]:
-            print tname
-
-        if fo.currentTurn() == 1:
-            return
-        if foAI.foAIstate.aggression <= fo.aggression.cautious:
-            research_queue_list = get_research_queue_techs()
-            def_techs = TechsListsAI.defense_techs_1()
-            for def_tech in def_techs:
-                if def_tech not in research_queue_list[:5] and not tech_is_complete(def_tech):
-                    res = fo.issueEnqueueTechOrder(def_tech, min(3, len(research_queue_list)))
-                    print "Empire is very defensive, so attempted to fast-track %s, got result %d" % (def_tech, res)
-        if False and foAI.foAIstate.aggression >= fo.aggression.aggressive:  # with current stats of Conc Camps, disabling this fast-track
-            research_queue_list = get_research_queue_techs()
-            if "CON_CONC_CAMP" in research_queue_list:
-                insert_idx = min(40, research_queue_list.index("CON_CONC_CAMP"))
-            else:
-                insert_idx = max(0, min(40, len(research_queue_list)-10))
-            if "SHP_DEFLECTOR_SHIELD" in research_queue_list:
-                insert_idx = min(insert_idx, research_queue_list.index("SHP_DEFLECTOR_SHIELD"))
-            for cc_tech in ["CON_ARCH_PSYCH", "CON_CONC_CAMP"]:
-                if cc_tech not in research_queue_list[:insert_idx + 1] and not tech_is_complete(cc_tech):
-                    res = fo.issueEnqueueTechOrder(cc_tech, insert_idx)
-                    msg = "Empire is very aggressive, so attempted to fast-track %s, got result %d" % (cc_tech, res)
-                    if report_adjustments:
-                        chat_human(msg)
-                    else:
-                        print msg
-
-    elif fo.currentTurn() > 100:
-        generate_default_research_order()
 
-    research_queue_list = get_research_queue_techs()
-    num_techs_accelerated = 1  # will ensure leading tech doesn't get dislodged
-    got_ggg_tech = tech_is_complete("PRO_ORBITAL_GEN")
-    got_sym_bio = tech_is_complete("GRO_SYMBIOTIC_BIO")
-    got_xeno_gen = tech_is_complete("GRO_XENO_GENETICS")
     #
-    # Consider accelerating techs; priority is
-    # Supply/Detect range
-    # xeno arch
-    # ast / GG
-    # gro xeno gen
-    # distrib thought
-    # quant net
-    # pro sing gen
-    # death ray 1 cleanup
-
-    nest_tech = AIDependencies.NEST_DOMESTICATION_TECH
```

Comment:
It looks like the handling of domestication tech was not carried over in any way, which should be rectified.